src/xunit.performance.api/XunitRunner.cs

```diff
@@ -70,7 +72,25 @@ private static void SetupRunnerCallbacks(AssemblyRunner runner, ManualResetEvent
             {
                 lock (consoleLock)
                 {
-                    WriteInfoLine($"Running {info.TestCasesToRun} Benchmarks out of {info.TestCasesDiscovered} Xunit Facts...");
+                    var diff = info.TestCasesDiscovered - info.TestCasesToRun;
+                    if (diff < 0) // This should never happen.
+                    {
+                        // TODO: Add error handling. Throwing exception is not enough because we are waiting for the async runner to finish.
+                        WriteErrorLine("There are more benchmarks than facts.");
+                    }
+                    var message = (diff == 0) ? $"Running {info.TestCasesToRun} Benchmarks..." : $"Running {info.TestCasesToRun} Benchmarks out of {info.TestCasesDiscovered} Xunit Facts...";
```

Comment:
I think this would be easier to interpret for the `diff != 0` case:
`$"Running {info.TestCasesToRun} Benchmarks, skipping {diff} Xunit Facts..."`